### PR TITLE
Paper-cut command added

### PR DESCRIPTION
**Pull request by Blair2004**
- Added : Paper-cut commands

### CODE REVIEW

File: lib/buffer-builder.ts [hunk 1]
```diff
@@ -150,6 +150,16 @@ export class BufferBuilder {
     return this.buffer.flush();
   }
 
+  /**
+   * Register Paper Cut Action
+   * @return void
+   */
+  public paperCut(): BufferBuilder {
+    this.buffer.write( Command.GS_v(1) );
```
Comment (ingoncalves) [inline lib/buffer-builder.ts]: Please, remove the white-spaces inside ( );

Comment (Blair2004) [reply lib/buffer-builder.ts]: ok

**Comment by Blair2004** [comment]
Done

**Comment by Blair2004** [comment]
What do you mean by reset package-lock.json ?

**Pull request merged**